docs/migration.md

```diff
@@ -0,0 +1,421 @@
+ACPCore.extensionVersion(): Promise<string>
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#extensionversion)
+```javascript
+MobileCore.extensionVersion(): Promise<string>
+```
+
+#### getLogLevel
+- ACP (2.x)
+```javascript
+ACPCore.getLogLevel(): Promise<string> 
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#getloglevel)
+```javascript
+MobileCore.getLogLevel(): Promise<string> 
+```
+
+#### getSdkIdentities
+- ACP (2.x)
+```javascript
+ACPCore.getSdkIdentities(): Promise<?string>
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#getsdkidentities)
+```javascript
+MobileCore.getSdkIdentities(): Promise<?string>
+```
+
+#### getPrivacyStatus
+- ACP (2.x)
+```javascript
+ACPCore.getPrivacyStatus(): Promise<string>
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#getprivacystatus)
+```javascript
+MobileCore.getPrivacyStatus(): Promise<string>
+```
+
+#### log
+- ACP (2.x)
+```javascript
+ACPCore.log(logLevel: string, tag: string, message: string)
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#log)
+```javascript
+MobileCore.log(logLevel: string, tag: string, message: string)
+```
+
+#### resetIdentities
+- ACP (2.x)
+```javascript
+ACPCore.resetIdentities()
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#resetidentities)
+```javascript
+MobileCore.resetIdentities()
+```
+
+#### setPrivacyStatus
+- ACP (2.x)
+```javascript
+ACPCore.setPrivacyStatus(privacyStatus: string) 
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#setprivacystatus)
+```javascript
+MobileCore.setPrivacyStatus(privacyStatus: string) 
+```
+
+#### setLogLevel
+- ACP (2.x)
+```javascript
+ACPCore.setLogLevel(mode: string)
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#setloglevel)
+```javascript
+MobileCore.setLogLevel(mode: string)
+```
+
+#### updateConfiguration
+- ACP (2.x)
+```javascript
+ACPCore.updateConfiguration(configMap?: { string: any })
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#updateconfiguration)
+```javascript
+MobileCore.updateConfiguration(configMap?: { string: any })
+```
+
+### Identity
+
+#### appendVisitorInfoForURL
+- ACP (2.x)
+```javascript
+ACPCore.appendVisitorInfoForURL(baseURL?: String): Promise<?string> 
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#appendvisitorinfoforurl)
+```javascript
+MobileCore.appendVisitorInfoForURL(baseURL?: String): Promise<?string> 
+```
+
+#### extensionVersion
+- ACP (2.x)
+```javascript
+ACPCore.extensionVersion(): Promise<string>
+```
+- [AEP (1.x)](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#extensionversion-1)
+```javascript
+MobileCore.extensionVersion(): Promise<string>
```

Comment:
Why extenstionVersion is under Identity?

Reply:
It's updated to ->
```javascript
ACPIdentity.extensionVersion(): Promise<string>
```